compiler/rustc_mir/src/interpret/eval_context.rs

```diff
@@ -824,12 +855,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         }
 
         if !self.stack().is_empty() {
-            info!(
-                "CONTINUING({}) {} (unwinding = {})",
-                self.frame_idx(),
-                self.frame().instance,
-                unwinding
-            );
+            info!(unwinding);
```

Comment:
Why is `info!(unwinding)` run twice? Also just having `unwinding=true/false` in the log without much context is not very clear...

Reply:
The duplication is gone, and we now inform about the stack popping again

Ooh, maybe we should try out `#[instrument]` next :laughing:

Reply:
> Ooh, maybe we should try out #[instrument] next laughing

No idea what that means.^^